content/en/developers/integrations/agent_integration.md

```diff
@@ -440,6 +440,33 @@ sudo datadog-agent integration install -w /path/to/wheel.whl
 
 Once you have created your Agent-based integration, see the [Create a tile][20] documentation for information on populating the remaining [required assets][31] that appear on your integration tile, and opening a pull request.
 
+## Update your integration
+To update your integration, edit the relevant files and open a new pull request to your integration's directory in the [`integrations-extras`][21] or [`marketplace`][22] repository. 
+
+* If you are editing or adding new integration code, a version bump is required.
+
+* If you are editing or adding new README content, manifest information, or assets such as dashboards and recommended monitors, a version bump is optional. 
```

Comment:
Nit: version bumps for assets and README are unnecessary, they would be bumping the version but nothing would change. I think we should say these changes don't need a version bump instead of being optional.

Reply:
+1

Reply:
Gotcha, will change this